app/src/processing/app/syntax/JEditTextArea.java

```diff
@@ -189,6 +190,7 @@ protected TextAreaPainter createPainter(final TextAreaDefaults defaults) {
   /**
    * Inline Input Method Support for Japanese.
    */
+  @Override
```

Comment:
I will take care of the annotations problem...i guess i missed one or two...i will check it out